components/Mentor/index.jsx

```
@@ -45,59 +107,108 @@ export const Mentor = () => {
   const closePopup = () => setShowPopup(false);
 
   const onSubmit = async (values, { resetForm }) => {
-    setLoading(true);
+
+    { !dataAccepted && setShowDataModal(true) }
+
     setIsSubmitting(true);
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)